hugo/content/3_how-tos/34_networking-and-communication/341_creating-ros-networks.md

```diff
@@ -37,7 +37,7 @@ To create a routed/native network:
 As Kinetic has reached the EOL, it is suggested to select **Melodic** as the ROS version while creating a build.
 {{%/notice%}} |
 |**Runtime**| Select the runtime as **Device** or **Cloud**. If you create a device runtime, ensure you have a rapyuta.io registered device with docker runtime and AMD64 architecture available.<br> If you select the device runtime: <br> <ul><li> Select the **Device** and its **IP Interface** from the dropdown list.</li> <li> Select the device restart policy. </li></ul> |
-|**Resource limit** (applicable only for cloud runtime)| Select the memory allocation and computational ability of the routed network from the dropdown list. These resources are reserved in the platform for effective ROS communication. You can select the resource limits based on your requirement.|
+|**Resource limit** (applicable only for cloud runtime)|  Select the CPU and memory requirement of the network from the dropdown list. These resources are reserved in the platform for effective ROS communication. You can also choose or enter the custom resource limits. For example, 1.025 vCPU and 2.5 GiB Memory.  <br> **Upper limit for all types of cloud networks:**<br>   * MaxCloudNetworkCPU: 4 <br>  * MaxCloudNetworkMemory: 16384 <br> **Lower limit for cloud routed networks:** <br>  * MinCloudRoutedNetworkCPU: 0.5 <br>  * MinCloudRoutedNetworkMemory: 1024 <br> **Lower limit for cloud native networks:** <br>   * MinCloudNativeNetworkCPU: 0.1 <br>  * MinCloudNativeNetworkMemory: 512 <br> **Granularity:** <br>  * CPU Granularity: 0.025 cores <br>   * Memory Granularity: 128 MB <br> |
```

Comment:
This looks good to me. However, there's a section later in this file that talks about creating networks and the available limit options small, x-small, etc. It's currently not in line with this change and we'll have to wait for the UI change to go, I guess.

Reply:
will verify the UI and track it in a separate PR.